fix(tem): add information (#1683)
scaleway/docs-content: managed-services/transactional-email/how-to/manage-email-activity.mdx

5. Optionally, click **Reset filters** when you are done filtering your email activity.
   <Lightbox src="scaleway_reset_filters.webp" alt="" />

<Message type="note">

>  If you are using Transactional Email's API, you can [filter your emails via the API](https://www.scaleway.com/en/developers/api/transactional-email/#path-statistics-email-statuses)

Reviewer: ```suggestion
You can also [filter your emails via the Transactional Email API](https://www.scaleway.com/en/developers/api/transactional-email/#path-statistics-email-statuses)
```
